test/options/lines-between-rulesets-css/test.js

```diff
@@ -0,0 +1,24 @@
+/* global describe, it */
+'use strict';
+
+describe('options/lines-between-rulesets:css', function() {
```

Comment:
Yeah it does you're correct. However this only tests css, what about less? A test to check for nested selectors on the same line would be useful.